src/index.js

```diff
@@ -327,24 +332,24 @@ export default class Resizable extends React.Component<ResizableProps, State> {
     minWidth = typeof minWidth === 'undefined' ? undefined : Number(minWidth);
     minHeight = typeof minHeight === 'undefined' ? undefined : Number(minHeight);
 
-    const ratio = original.height / original.width;
+    const ratio = typeof lockAspectRatio === 'number' ? lockAspectRatio : original.width / original.height;
```

Comment:
Note that I reversed the order here. It is now `width / height` instead of `height / width`. I did this because aspect ratios are normally calculated as width to height (such as 16:9). By doing this, all of the ratio calculations below had to swap `*` for `/` and vice-versa.